aten/src/ATen/native/xnnpack/RegisterOpContextClass.cpp

```diff
@@ -62,9 +64,9 @@ TORCH_LIBRARY(xnnpack, m) {
               std::move(std::get<3>(state)),
```

Comment:
Not familiar with this. Why we remove std::move for these? Can add a comment?

Reply:
This is actually suggested by the internal lint. Something like `std::move` does not have any effect when we are returning a `const`.

Reply:
ok, can we remove all of the "move"? Seems we didn't treat all the fields uniformly.